diff --git a/build/linux/debian/calculate-deps.js b/build/linux/debian/calculate-deps.js
index 6304df9..202ddcf 100644
--- a/build/linux/debian/calculate-deps.js
+++ b/build/linux/debian/calculate-deps.js
@@ -79,9 +79,11 @@ function calculatePackageDeps(binaryPath, arch, chromiumSysroot, vscodeSysroot)
     // TODO(deepak1556): remove this workaround in favor of computing the
     // versions from build container for native modules.
     const filteredDeps = depsStr.split(', ').filter(dependency => {
-        return !dependency.startsWith('libgcc-s1');
+		return !dependency.startsWith('libgcc-s1') &&
+			!dependency.startsWith('libgssapi-krb5-2') &&
+			!dependency.startsWith('libkrb5-3') &&
+			!dependency.startsWith('libxkbfile1');
     }).sort();
     const requires = new Set(filteredDeps);
     return requires;
 }
-//# sourceMappingURL=calculate-deps.js.map
\ No newline at end of file
+//# sourceMappingURL=calculate-deps.js.map
